src/models/EET.ts

```diff
@@ -6,13 +6,14 @@
  * @year 2022
  */
 
-import axios from "axios";
-import { GoPay } from "..";
-import { eet } from "../types/eet";
+import { GoPay } from "~/goPay";
+import { eet } from "~/types/eet";
 
 export class Misc {
+  // @ts-ignore
   private __client: GoPay;
-    private __sufix = "/payments/payment";
+  // @ts-ignore
+  private __sufix = "/payments/payment";
```

Comment:
Isn't this a unsed variable?

Reply:
Yep, it is. I would delete it too, but I don't know why it got into the master in the first place. I guess it was for some reason. That is why I left it there.